server/engine/Game.js

```diff
@@ -0,0 +1,124 @@
+const wordArray = require('./Words');
+
+class Game {
+  ADD_POINT = 100;
+
+  /**
+   * @param {string} id : Room Name 
+   * @param {array} players : game players array
+   */
+  constructor(id, players) {
+    this.roomName = id;
```

Comment:
You don't have to store roomName here. We will have **roomName** with a **Game** object in our key-value cache.